.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [Optional ] Please Branch should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section')